salt/iiif/loris.sls

```diff
@@ -42,31 +46,36 @@ loris-dependencies:
             - libfreetype6
             - libfreetype6-dev
             - zlib1g-dev
+            {% if osrelease == "14.04" %}
             - liblcms
             - liblcms-dev
             - liblcms-utils
+            - libtiff4-dev
+            {% endif %}
+
             - liblcms2-2 
             - liblcms2-dev 
             - liblcms2-utils
-            - libtiff4-dev
             - libtiff5-dev
             - libxml2-dev
             - libxslt1-dev
 
+    # TODO: these requirements best live in a requirements.txt file where we can get security feedback
```

Comment:
I wish, it took days to get this installation process right. We should switch to a [container image](https://github.com/bodleian/loris-grok-docker) before trying to improve this.

Reply:
sure. iiif needs to be upgraded as well. It could all be bundled together

Reply:
of course not, one update at a time it's safer and reversible :nerd_face:

Reply:
> It could all be bundled together

not into *this* commit, obviously ;)